cmd/http/main.go

```diff
@@ -61,7 +66,21 @@ func main() {
 	}
 	defer nc.Close()
 
-	conn, err := grpc.Dial(cfg.ThingsURL, grpc.WithInsecure())
+	secureOption := grpc.WithInsecure()
+	if cfg.CACerts != "" {
+		thingsURL, err  := url.Parse(cfg.ThingsURL)
+		if err != nil {
+			logger.Error(fmt.Sprintf("Failed to parse ThingsURL: %s", err))
+		}
+		tpc, err := credentials.NewClientTLSFromFile(cfg.CACerts, thingsURL.Host)
+		if err != nil {
+			logger.Error(fmt.Sprintf("Failed to load certs: %s", err))
+			os.Exit(1)
+		}
+		secureOption = grpc.WithTransportCredentials(tpc)
+	}
```

Comment:
@nwest1 I think, It would be good if in case that no CA certs are set, to log a warning so that the user knows that the gRPC communication is not encrypted.

Reply:
Updated!

Reply:
@nwest1 cool, but I personally think logger.Warn might be more appropriate.

@drasko, @dusanb94 what do you think about this?

Reply:
All the other `logger.Warn`s are currently related to some error happening, but the app will continue to run. Though I think security is important, it's not in error, which is why I chose info. 

Happy to change this to whatever makes sense for the project.

Reply:
@nwest1 I agree with you, we should use warn only for errors. This is valid flow, so I think that we should leave info.

Reply:
@nwest1 thanks for checking on other warning usages. I totally agree with you. Let's leave it as an info message.